code/modules/virus2/effect/effect.dm

```diff
@@ -461,22 +463,47 @@
 
 /datum/disease2/effect/butterfly_skin
 	name = "Epidermolysis Bullosa"
-	stage = 2
+	stage = 1
```

Comment:
Diseases progress from stages 1 to 4 both in the code and in-game examination - I think you meant this to be stage 4?